packages/db/fauna/resources/Index/uploadByUserAndContent.js

```diff
@@ -0,0 +1,52 @@
+import fauna from 'faunadb'
+
+const {
+  CreateIndex,
+  Collection,
+  If,
+  Index,
+  Exists,
+  Not,
+  Query,
+  Lambda,
+  Equals,
+  Select,
+  Var
+} = fauna
+
+/**
+ * Usage:
+ *
+ * Match(
+ *   Index('upload_by_user_and_content'),
+ *   Ref(Collection('user'), Var('id')),
+ *   Ref(Collection('content'), Var('id')),
+ *   true // not_deleted
```

Comment:
would be lovely to not have to remember to pass an unnamed boolean to ensure you dont count deleted things which is mostly what you want